src/components/Shell/QRCodeDialog.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  IconButton,
+} from '@mui/material'
+import React from 'react'
+import CloseIcon from '@mui/icons-material/Close'
+import { QRCode } from 'react-qrcode-logo'
+
+const QR_CODE_SIZE = 256
+const QR_IMAGE_OPACITY = 0.3
+
+export interface QRCodeDialogProps {
+  isOpen: boolean
+  handleClose: () => void
+}
+
+export function QRCodeDialog({ isOpen, handleClose }: QRCodeDialogProps) {
+  const url = window.location.href
+  return (
+    <Dialog
+      open={isOpen}
+      onClose={handleClose}
+      aria-labelledby="alert-dialog-title"
+      aria-describedby="alert-dialog-description"
+    >
+      <DialogTitle id="alert-dialog-title">
+        {'Room QR Code'}
```

Comment:
Is there a specific reason this string is quoted, or could it be printed raw here?

```suggestion
        Room QR Code
```

Reply:
Addressed